lib/interfaces/plugins.js

```diff
@@ -300,6 +300,13 @@ module.exports = function (app) {
       })
     }
 
+    appCopy.registerToTransmitPGNs = (id, pgns) => {
```

Comment:
I think it should be server's responsibility to keep track of the plugin id here, and not allow a plugin to use arbitrary id's to register. As it is this allows plugins to overwrite other plugins' data. 

Or is there a use case where it would make sense to allow that?